packages/sdk/react-shell/src/playwright/invitations-manager.ts

```diff
@@ -150,7 +150,7 @@ export class InvitationsManager extends ShellManager {
       await peer.getByTestId('join-identity').click();
     }
     await peer.getByTestId(`${type === 'device' ? 'halo' : 'space'}-invitation-input`).waitFor({ timeout: 500 });
-    await peer.getByTestId(`${type === 'device' ? 'halo' : 'space'}-invitation-input`).type(invitation);
```

Comment:
may be worth taking a peek at the shell manager as well to see if it's using `type`, perhaps that causes some flakiness there too

Reply:
I looked for other uses of `.type` when I found this, but didn't find any.